source/_docs/drush.md

```diff
@@ -364,7 +364,8 @@ See the [Drush Migrate documentation](https://drupal.org/node/1561820) for detai
 
 ## Known Limitations
 - Crontab: Currently, there is no way to manage Crontab on Pantheon. If you need a way to set up your own Cron interval, you can use an external cron service such as [Easy Cron](https://www.easycron.com/user/register).
-- The following Drush commands are not supported and will not work on Pantheon sites: `sql-sync-pipe`, `sql-sqlc`, `php-eval`. An alternative to `drush sql-sync` is to use `drush sql-dump` instead.
+- The `sql-sync-pipe` and `sql-cli` (`sqlc`) Drush commands are not supported and will not work on Pantheon sites. An alternative to `drush sql-sync` is to use `drush sql-dump` instead.
+- The `php-eval` (`eval`, `ev`) command only partially works on Pantheon. Pantheon rejects any incoming command-line request that has an argument or option that contains characters such as `;` or `$`. As an example, `drush @pantheon.site.env ev 'return foo()` works, because Drush provides a trailing `;` if needed.

```

Comment:
I'd rather direct users to using `drush php-script` instead. The degree to which `php-eval` is impeded makes it functionally useless, and it will only frustrate people vs having them write a script and execute it.